src/puppetlabs/dujour/version_check.clj

```diff
@@ -26,13 +26,15 @@
     map?    ProductCoords))
 
 (def RequestValues
-  {:product-name ProductName
+  {(schema/optional-key :certname) schema/Str
+   (schema/optional-key :cacert) schema/Str
```

Comment:
I think this was mentioned in the channel yesterday, but we should rename these to something like:

- `host-id` (or maybe `master-id`)
- `site-id`

Thoughts @pcarlisle ?